oracle.py

```diff
@@ -68,97 +74,106 @@ def update_dataset(self, queries, energies):
 
 
 class OracleBase:
-    '''
+    """
     BaseClass model for all other oracles
-    '''
-    def __init__(self, config):
+    """
+
+    def __init__(self, config, logger):
         self.config = config
+        self.logger = logger
```

Comment:
self.logger is not used in the OracleBase derived classes for now.
It is only used in the generic scoring function, right ? 
What whas the thought behind passing self.logger to OracleBase ? ;)

Reply:
Yep, you're right. We don't need to pass it to OracleBase. I'll delete that